packages/content-conformance/src/rules/ensure-valid-nav-paths.js

```diff
@@ -0,0 +1,71 @@
+import flat from 'flat'
+
+// Recursively traverse json nav data, and validate node paths against flat list of files from context
+function visitor(navData, filePaths, report) {
+  // flatten nav data file
+  const flatNodes = flat(navData)
+  // get all the paths from the nav data file
+  const navDataPaths = Object.entries(flatNodes)
+    .filter(([key]) => {
+      return key.endsWith('.path')
+    })
+    .map(([, value]) => value)
```

Comment:
[question] Did the old validation check relative `href` values? e.g. in `docs-nav-data` you could have `href: "/api-docs/foo/bar"`

Reply:
Nope, the [old validation](https://github.com/hashicorp/react-components/blob/main/packages/docs-sidenav/utils/validate-file-paths/index.ts) ignores `href` fields

On a related note of different fields:
I might need to handle the `remoteFile` field, but I couldn't actually find an example of nav-data that had this field:
- https://github.com/hashicorp/react-components/blob/4cfd21a5c186c1140a6d40abc5776b5c8d1e642c/packages/docs-sidenav/utils/validate-file-paths/index.ts#LL18C3-L18C41

Reply:
That's a packer plugin specific thing. I believe it gets populated when the plugin content is fetched during build. It should be safe to ignore for now 💭